packages/auto-dag-data/src/encryption/index.ts

```diff
@@ -0,0 +1,47 @@
+import { Crypto } from '@peculiar/webcrypto'
+import { asyncByChunk } from '../utils/async.js'
+
+const crypto = new Crypto()
+
+export const ENCRYPTING_CHUNK_SIZE = 1024 * 1024
+const IV_SIZE = 16
+const TAG_SIZE = 16
+const ENCRYPTED_CHUNK_SIZE = ENCRYPTING_CHUNK_SIZE + IV_SIZE + TAG_SIZE
+
+const getKeyFromPassword = async (password: string) => {
```

Comment:
Is there a reason to not salt the password?

Reply:
Not a good one, I just copied from the PoC. We should enable salting.

Reply:
Thanks for the catch

Reply:
Done